gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -87,6 +85,7 @@ class {{ service.client_name }}Meta(type):
     {% endif %}
     {% if "rest" in opts.transport %}
     _transport_registry["rest"] = {{ service.name }}RestTransport
+    {# TODO(#2121): Remove this condition when async rest is GA. #}
```

Comment:
nit nit: Consider having the full URL in the TODO just in case we search for full URL and don't think to search just for the issue number. Same elsewhere.